### ISSUE TITLE
Localization Support

### ISSUE THREAD

**Issue by yugimani** [kerala-biodiversity/iNaturalist-Kerala#2]
iNat Web & Mobile Apps should support Dravidian Languages for better reach.

**Comment by manojkmohan** [comment]
Yes. Its Truly needed feature and This is a opportunity to contribute iNaturalists Technology for overcome the language barrier and local popularization.
Here is a help page found https://www.inaturalist.org/pages/translate
Upstreem Locales https://github.com/inaturalist/inaturalist/tree/master/config/locales
Mobile App https://github.com/inaturalist/iNaturalistAndroid/tree/main/iNaturalist/src/main/res

Checking this no much indian language in the list. in quick check Marathi & Hindi found.

crowdin.com this service is using for the localisation. so we need to start from here.

**Comment by manojkmohan** [comment]
Did a request to enable the languages (ml & ta). 
https://crowdin.com/project/inaturalistweb/discussions/48
Please join in the website and do support in the request. 
we can ask helps from @smc if needed. @joicemjoseph is manager of mozilla malayalam team.
@firosak @yugimani @pythianjoseph 

its need to cross 75% get implement in system. we can try to mobilize a monthly series of a weekend for some virtual translating sprints.

**Comment by yugimani** [comment]
iNat has approved our collective demand and enabled both Malayalam and Tamil.

1. *Mobile App Translation:* 
  a. https://crowdin.com/project/inaturalistios/ml-IN#
  b. https://crowdin.com/project/inaturalistios/ta#
2. *Web Translation:* 
  a. https://crowdin.com/project/inaturalistweb/ml-IN#
  b. https://crowdin.com/project/inaturalistweb/ta#

Translating to regional language will help democratize citizen-science. We need community engagement to translate. Feel free to spread the word or contact me and Manoj, if you have any questions.

**Comment by manojkmohan** [comment]
Here is some useful link can check in l10n activities

A Terminology corpus from Fuel Project http://fuelproject.org/contribute/index.php?route=search  
If we getting some grip, we can try for a biodiversity related terminology corpus in future.

www.olam.in a simple en - ml dictionary. 

https://translatewiki.net its similor project like crowdin, but using mediawiki framework and there are lot project we can refer including Wikipedia interface
